packages/react-relay/relay-hooks/__tests__/useBlockingPaginationFragment-test.js

```diff
@@ -2491,7 +2490,6 @@ describe('useBlockingPaginationFragment', () => {
         // the component twice: `expectFragmentResults` will fail in the next
         // test
         jest.resetModules();
-        disallowWarnings();
```

Comment:
Cannot nest this function within a test, because it contains an internal call to `afterEach`.